### DISCUSSION TITLE
Privacy manifest for AppStore releases

### DISCUSSION THREAD

**Discussion by am-apu** [pointfreeco/swift-snapshot-testing#847] [General]
Hey there, starting from 1st of may, its necessary to provide the privacy manifest (https://developer.apple.com/documentation/bundleresources/privacy_manifest_files)
Do you guys know, if we need this here too?

Bests

**Comment by mbrandonw** [comment]
Hi @am-apu, please use the Discussions tab in this repo for asking questions. This is not an issue with the library.

Also, we don't think a privacy manifest file is necessary since ideally you should not be including this code in your app target. It is only a testing library.

**Reply by am-apu** [reply]
i was tapping "adding a question" - sorry for that. would have expected its doing its way to publish this as a discussion.
Okay thank you - was not sure if this is necessary or not. Have a great day!
